packages/appeals-service-api/src/spec/appellant-submission.yaml

```diff
@@ -97,6 +97,8 @@ components:
 
         siteAreaSquareMetres:
           type: integer
+        tenantAgriculturalHolding:
+          type: boolean
```

Comment:
If you cd into the appeals-service-api and run 'npm run gen-api-types' it will update the api-types.d.ts from the yaml file

Reply:
Thank you, good to know for future!